crates/bindings/c/src/types.rs

```diff
@@ -181,3 +188,48 @@ impl<'a> From<&'a mut libsql_rows_future<'a>> for libsql_rows_future_t<'a> {
         Self { ptr: value }
     }
 }
+pub struct libsql_row {
+    pub(crate) result: libsql::Row,
+}
+
+#[derive(Clone, Debug)]
+#[repr(transparent)]
+pub struct libsql_row_t {
+    ptr: *const libsql_row,
+}
+
+impl libsql_row_t {
+    pub fn null() -> libsql_row_t {
+        libsql_row_t {
+            ptr: std::ptr::null(),
+        }
+    }
+
+    pub fn is_null(&self) -> bool {
+        self.ptr.is_null()
+    }
+
+    pub fn get_ref(&self) -> &libsql::Row {
+        &unsafe { &*(self.ptr) }.result
+    }
+
+    #[allow(clippy::mut_from_ref)]
+    pub fn get_ref_mut(&self) -> &mut libsql::Row {
+        let ptr_mut = self.ptr as *mut libsql_row;
+        &mut unsafe { &mut (*ptr_mut) }.result
+    }
+}
+
+#[allow(clippy::from_over_into)]
```

Comment:
Is this `#[allow(clippy::from_over_into)]` directive necessary at all? If I remember correctly it warns if you implement `Into` trait instead of `From`, which is more versatile, while you correctly implement `From` here. Maybe it's out-of-date?

Reply:
done